frontend/sozisel-app/src/components/ParticipantActiveSession/Modules/WhiteboardEvent/ParticipantWhiteBoard.scss

```diff
@@ -0,0 +1,62 @@
+@import "../../../../styles/styles";
+
+
+.ParticipantWhiteboardEvent {
+    display: flex;
+    flex-direction: column;
+    width: calc(100% - 2*#{$spacing-04});
```

Comment:
Can't all be done with flexes and margins? You together with @jmysliv use this ugly calculate because you don't seem to find a better way to style those...